src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1454,7 +1455,18 @@ export class AmpStoryPlayer {
     this.touchEventState_.startX = coordinates.screenX;
     this.touchEventState_.startY = coordinates.screenY;
 
-    this.pageScroller_.onTouchStart(event.timeStamp, coordinates.clientY);
+    this.pageScroller_ &&
+      this.pageScroller_.onTouchStart(event.timeStamp, coordinates.clientY);
+
+    this.element_.dispatchEvent(
+      createCustomEvent(
+        this.win_,
+        'amp-story-player-touchstart',
+        dict({
+          'coordinates': coordinates,
```

Comment:
Isn't it very unconventional to pass custom data here, over the default `touch*` event data? It would be much easier to pass the real event data, instead of creating custom data that we have to document.